docs/user/BuildLocally.md

```diff
@@ -10,6 +10,10 @@ During initial setup or if you have a new machine, run this script:
 ``` shell
 # either run as root or use sudo
 ./etc/DependencyInstaller.sh
+
+Options:    ./etc/DependencyInstaller.sh -prefix=DIR    # installs dependencies in an existing user specified directory
+            ./etc/DependencyInstaller.sh -local         # installs common dependencies in "$HOME/.local". Root access is still needed for other
+                                                        # dependencies. Flag cannot be used with sudo or with root access
```

Comment:
Remove to avoid duplicated info. It is ok to have only line 12 as it will likely be the default path. If a user has special requirements such as using `-local` and `-prefix` options they can access their description via the top level docs `Build.md` where we have all the options listed.